os-config.tpl.yml

```diff
@@ -150,6 +150,7 @@ rancher:
       - /usr/bin/iptables:/sbin/iptables:ro
       - /media:/media:shared
       - /mnt:/mnt:shared
+      - /:/host
```

Comment:
Not that it matters much for just the installer container, but you could probably bind mount  `/dev:/host/dev` and run `ros entrypoint` as the entrypoint instead of `/:/host`.